### ISSUE TITLE
ChatControl 11.0.3-BETA: IP Links broken in latest version

### ISSUE THREAD

**Issue by cades66** [kangarko/ChatControl#2932]
### Are you using MySQL?

No

### Are you using a proxy?

No

### "/version ChatControlRed" - plugin version

11.0.3-BETA

### ZIP of "/chc debug"

https://ufile.io/c57f1udv

### Steps to reproduce

In command.rs, use the following command as an example:

```
# /worldmap command
match ^[/]worldmap$
#dont verbose
then warn {prefix_info} View a live world-map by clicking here: &bhttp://{IP ADDRESS}:8038/
then deny
```

What's expected is a clickable IP - this was working in ChatControl 10.
Instead, in ChatControl 11, it is unclickable.

**Comment by kangarko** [comment]
Yep, you now need to use the proper minimessage syntax like this:

`View a live world-map by clicking here: &b<click:open_url:'http://{IP ADDRESS}:8038/'>http://{IP ADDRESS}:8038/</click>`

Use https://webui.advntr.dev/ for help.

We previously relied on an unsupported functionality by bungeecord chat API which we have now abandoned in favor of Adventure which does not do the url click translation automatically
